### ISSUE TITLE
Server-wide events screen

### ISSUE THREAD

**Issue by flambard-took** [mangband/mangband#1252]
First off, I really hate the new death message (#1159), as it's long and hard to parse. Not to mention it often truncates the *most interesting* piece of info -- the cause of death.

(Real world example: "- The brave hero ZalPal the level 47 Dwarf Paladin was killed by a Great Wyrm of B")

I also think it's not here nor there, i.e. we're not reporting similar things for slain uniques (arguably, it would make sense, "the brave hero X, the level 12 Human Warrior, has slain Grip, Farmer Maggot's Dog"). Also, I would've liked to see the depth for each such event ("was killed by Potion of Death, at 2400 feet").

So here is what I propose:

A new knowledge screen, very similar to Character History screen, but that would keep track of *all* the players. We can add interesting info to it, character levels, race / class, dungeon depth, turn it happened, the message itself, etc. Since each message would probably be formatted to take multiple lines, we won't be hitting any message-length limits.

I'm also imagining it would be limited to 20-40 most recent entries.

And once this is done, I'll hopefully be allowed to revert #1159 :)

**Comment by igroglaz** [comment]
this is awesome idea. I suppose that if it would be possible to add such messages to discord at separate channel - it would be wonderful, as discord stores all history without wiping it and it could be interesting to check some past events there.

maybe add 'Noteworthy Occurances' ad discord for particular such messages? (leaving them in main channel too) // yet another 'question' :)
